Formula/habitus.rb

```diff
@@ -0,0 +1,17 @@
+class Habitus < Formula
+  desc "A Build Flow Tool for Docker"
+  homepage "http://www.habitus.io"
+  url "https://github.com/cloud66/habitus/releases/download/1.0.1/habitus_darwin_amd64"
+  version "1.0.1"
```

Comment:
Do we need to explicitly define the `version`? It should normally be inferable by the `url`.

Reply:
It's confused.
```
Josephs-MacBook-Pro:~ joe$ brew irb
==> Interactive Homebrew Shell
Example commands available with: brew irb --examples
irb(main):001:0> Version.detect("https://github.com/cloud66/habitus/releases/download/1.0.1/habitus_darwin_amd64", {})
=> #<Version::FromURL:0x007fd85284b980 @version="64">
```
@cglong note `brew audit` will puke if an explicit version is redundant.